.evergreen/config.yml

```diff
@@ -802,6 +802,10 @@ functions:
           if [ "${SKIP_EC2_AUTH_TEST}" = "true" ]; then
             exit 0
           fi
+
+          # Write an empty prepare_mongodb_aws so no auth environment variables are set.
+          echo "" > "${PROJECT_DIRECTORY}/prepare_mongodb_aws.sh"
```

Comment:
Optional: `: > "${PROJECT_DIRECTORY}/prepare_mongodb_aws.sh"` also clears a file. Otherwise, `echo -n > "${PROJECT_DIRECTORY}/prepare_mongodb_aws.sh"` writes the file without an additional new line.

Reply:
I came across the same comment. I used to use `> file` but it failed on Mac.